plugins/database/redshift/redshift_test.go

```diff
@@ -100,29 +101,31 @@ func TestPostgreSQL_Initialize(t *testing.T) {
 
 	// Test decoding a string value for max_open_connections
 	connectionDetails = map[string]interface{}{
-		"connection_url":       url,
-		"max_open_connections": "5",
+		"connection_url":       connURL,
+		"max_open_connections": "73",
 	}
 
 	_, err = db.Init(context.Background(), connectionDetails, true)
```

Comment:
Shouldn't this be calling `Initialize` to test the `Database` interface? If so, you can use the `dbtesting` package under `github.com/hashicorp/vault/sdk/database/dbplugin/v5/dbtesting` to make this call (and other calls to the DB) smoother

Reply:
Thanks - TIL about the `dbtesting` package.